assets/xml/objects/object_bombf.xml

```diff
@@ -1,11 +1,14 @@
 ﻿<Root>
     <File Name="object_bombf" Segment="6">
-        <DList Name="object_bombf_DL_000340" Offset="0x340" /> <!-- Original name is "bm_leaf_model" -->
-        <DList Name="object_bombf_DL_000408" Offset="0x408" /> <!-- Original name is "bm_flower_model" -->
-        <DList Name="object_bombf_DL_000530" Offset="0x530" /> <!-- Original name is "bm_leaf2_model" -->
-        <Texture Name="object_bombf_Tex_0005D8" OutName="tex_0005D8" Format="rgba16" Width="32" Height="32" Offset="0x5D8" />
-        <Texture Name="object_bombf_Tex_000DD8" OutName="tex_000DD8" Format="rgba16" Width="16" Height="32" Offset="0xDD8" />
-        <Texture Name="object_bombf_Tex_0011D8" OutName="tex_0011D8" Format="ia16" Width="32" Height="32" Offset="0x11D8" />
-        <Texture Name="object_bombf_Tex_0019D8" OutName="tex_0019D8" Format="rgba16" Width="4" Height="4" Offset="0x19D8" />
+        <!-- Bomb Flower DisplayLists -->
+        <DList Name="gBombFlowerLeavesDL" Offset="0x340"/>
+        <DList Name="gBombFlowerBombAndSparkDL" Offset="0x408"/>
+        <DList Name="gBombFlowerBaseLeavesDL" Offset="0x530"/>
+        
+        <!-- Bomb Flower DisplayList Textures -->
+        <Texture Name="gBombFlowerLeafOutwardsTex" OutName="bomb_flower_leaf_outwards" Format="rgba16" Width="32" Height="32" Offset="0x05D8"/>
+        <Texture Name="gBombFlowerLeafUpwardsTex" OutName="bomb_flower_leaf_upwards" Format="rgba16" Width="16" Height="32" Offset="0x0DD8"/>
```

Comment:
```suggestion
        <Texture Name="gBombFlowerLeafOutwardsTex" OutName="bomb_flower_leaf_outwards" Format="rgba16" Width="32" Height="32" Offset="0x5D8"/>
        <Texture Name="gBombFlowerLeafUpwardsTex" OutName="bomb_flower_leaf_upwards" Format="rgba16" Width="16" Height="32" Offset="0xDD8"/>
```
A lot of Offsets throughout all these changes have had a leading 0 added, could you go through and make sure it's reverted to as it was without leading 0's?